console/src/components/Services/ApiExplorer/Analyzer/QueryAnalyzer.js

```diff
@@ -31,21 +39,26 @@ export default class QueryAnalyser extends React.Component {
         this.props.clearAnalyse();
       });
   }
+  renderAnalysisList() {
```

Comment:
Instead of creating render function, let's create a new file `AnalyzeList.tsx` and put it there.

Reply:
Changes have been committed!